tox.ini

```
@@ -22,10 +22,12 @@ commands =
 [testenv:lint]
 basepython = python3
 deps =
+  black
   flake8
   setuptools-rust
 whitelist_externals=cargo
 commands =
+  black --check {posargs} '../retworkx' '../tests'
```

Just to make it a bit more consistent with `cargo fmt` (so the diff shows up)

```suggestion
  black --check --diff {posargs} '../retworkx' '../tests'
```